odoo-shippable/scripts/build-image.sh

```diff
@@ -129,6 +129,18 @@ wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/ru
 sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
 sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
 
+# Install YouCompleteMe
+apt-get install cmake
+add-apt-repository ppa:jonathonf/vim -y
+apt update
+apt-get upgrade
```

Comment:
And, as a general principle, if you are going to update a base software (like vim) in this image, you should upgrade it in the base image, which is [Vauxoo/docker-ubuntu-base](https://github.com/Vauxoo/docker-ubuntu-base), specifically here:

https://github.com/Vauxoo/docker-ubuntu-base/blob/master/scripts/build-image.sh#L31

Doing this change here will leave inconsistent versions of vim throughout the base image and its children, which is a bad testing practice.